web/src/pages/error/ErrorPage.jsx

```diff
@@ -0,0 +1,8 @@
+export default function ErrorPage() {
```

Comment:
And there could series of error pages, 4xx, 5xx, having that previous error on the frontend is classified as 500 (server failure /system error )